exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -231,6 +232,17 @@ def handle_redirect(location)
           # TODO: figure out destination and reinitialize @http and @path
         end
 
+        def log_status(body)
+          status = Google::Rpc::Status.decode(body)
+          details = status.details.map do |detail|
+            klass_or_nil = ::Google::Protobuf::DescriptorPool.generated_pool.lookup(detail.type_name).msgclass
+            detail.unpack(klass_or_nil) if klass_or_nil
+          end.compact
+          OpenTelemetry.handle_error(message: "OTLP exporter received rpc.Status{message=#{status.message}, details=#{details}}")
```

Comment:
Not blocking, but do we want to add `status.code` as a label that we pass into the @metrics_reporter? 

https://cloud.google.com/tasks/docs/reference/rpc/google.rpc#google.rpc.Code

Reply:
Not really. The OTLP spec says it doesn't use `status.code`. We "MAY record it for troubleshooting purposes", but it seems a bit pointless. I can add it if you're curious to see what we get.

Reply:
zero curiosity, this can be resolved